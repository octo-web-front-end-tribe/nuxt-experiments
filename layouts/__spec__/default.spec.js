import {shallow} from '@vue/test-utils';
import Default from '../default.vue';

describe('Default layout', () => {
  it('should match snapshot', () => {
    // when
    const wrapper = shallow(Default);

    // then
    expect(wrapper.element).toBeDefined();
    expect(wrapper.element).toMatchSnapshot();
  });
});
