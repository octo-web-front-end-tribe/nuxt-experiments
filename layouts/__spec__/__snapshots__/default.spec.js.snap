// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout | default should match snapshot 1`] = `
<div>
  <nuxt />
</div>
`;
